pvlib/iotools/solargis.py

```diff
@@ -0,0 +1,171 @@
+"""Functions to retrieve and parse irradiance data from Solargis."""
+
+import pandas as pd
+import requests
+import io
+
+URL = 'https://solargis.info/ws/rest/datadelivery/request?'
```

Comment:
I suggest leaving the `?` off the URL itself.  Makes more sense to add that in when appending the API key onto the URL IMHO.

Reply:
@kandersolar Initially I would agree, but removing the question mark results in the code not working...